atomic_reactor/plugins/input_osv3.py

```diff
@@ -41,11 +61,10 @@ def run(self):
             self.plugins_json = os.environ['ATOMIC_REACTOR_PLUGINS']
         except KeyError:
             try:
-                self.plugins_json = os.environ['DOCK_PLUGINS']
+                user_params = os.environ['USER_PARAMS']
```

Comment:
don't we want to 1st try in line 61 actually USER_PARAMS, and in case it doesn't exists fallback to ATOMIC_REACTOR_PLUGINS?

Reply:
Fixed.